### ISSUE TITLE
Cannot use same schema in different endpoints

### ISSUE THREAD

**Issue by itookyourboo** [mts-ai/FastAPI-JSONAPI#20]
This code will raise KeyError:

```python
RoutersJSONAPI(
    path='/path1'
    schema=CommentSchema,
    ...
)

RoutersJSONAPI(
    path='/pah2'
    schema=CommentSchema,
    ...
)
```

Traceback:

```
/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py:409 2023-05-28 15:40:09,372 [ ERROR ] : Exception in ASGI application
Traceback (most recent call last):
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/uvicorn/protocols/http/h11_impl.py", line 404, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/uvicorn/middleware/proxy_headers.py", line 78, in __call__
    return await self.app(scope, receive, send)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 106, in __call__
    raise exc from None
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/uvicorn/middleware/debug.py", line 103, in __call__
    await self.app(scope, receive, inner_send)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/fastapi/applications.py", line 270, in __call__
    await super().__call__(scope, receive, send)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/starlette/applications.py", line 124, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/starlette/middleware/errors.py", line 184, in __call__
    raise exc
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/starlette/middleware/errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/starlette/middleware/cors.py", line 84, in __call__
    await self.app(scope, receive, send)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 75, in __call__
    raise exc
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/starlette/middleware/exceptions.py", line 64, in __call__
    await self.app(scope, receive, sender)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/starlette/routing.py", line 680, in __call__
    await route.handle(scope, receive, send)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/starlette/routing.py", line 275, in handle
    await self.app(scope, receive, send)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/starlette/routing.py", line 65, in app
    response = await func(request)
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/fastapi/applications.py", line 225, in openapi
    return JSONResponse(self.openapi())
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/fastapi/applications.py", line 200, in openapi
    self.openapi_schema = get_openapi(
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/fastapi/openapi/utils.py", line 423, in get_openapi
    definitions = get_model_definitions(
  File "/home/wignorbo/.pyenv/versions/wayprogress_3.9.13/lib/python3.9/site-packages/fastapi/utils.py", line 35, in get_model_definitions
    m_schema, m_definitions, m_nested_models = model_process_schema(
  File "pydantic/schema.py", line 590, in pydantic.schema.model_process_schema
  File "pydantic/schema.py", line 631, in pydantic.schema.model_type_schema
  File "pydantic/schema.py", line 258, in pydantic.schema.field_schema
  File "pydantic/schema.py", line 536, in pydantic.schema.field_type_schema
  File "pydantic/schema.py", line 933, in pydantic.schema.field_singleton_schema
KeyError: <class 'pydantic.main.CommentSchemaObjectJSONAPI'>
```

**Comment by mahenzon** [comment]
Should be fixed in 1.0.0, please check 

https://pypi.org/project/FastAPI-JSONAPI/

https://github.com/mts-ai/FastAPI-JSONAPI/releases/tag/1.0.0

**Comment by mahenzon** [comment]
please reopen if the problem persists
